scripts/run-mypy-changed.sh

```
@@ -0,0 +1,26 @@
+      files="$file"
+    fi
+  fi
+done
+
+if [ -z "$files" ]; then
+  exit 0
+fi
+
+# Convert newline-separated files to array and run mypy
+set -f  # disable globbing
+.venv/bin/mypy $(echo "$files" | sort -u)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)